src/Language/PureScript/TypeChecker/TypeSearch.hs

```diff
@@ -114,5 +117,7 @@ typeSearch
 typeSearch unsolved env st type' =
   let
     resultMap = Map.mapMaybe (\(x, _, _) -> checkSubsume unsolved env st type' x $> x) (P.names env)
+    (allLabels, solvedLabels) = accessorSearch unsolved env st type'
+    solvedLabels' = first (P.Qualified Nothing . P.Ident . ("_." <>) . P.prettyPrintLabel) <$> solvedLabels
```

Comment:
Let's just return `Text` instead of `Ident` here, if they're not all really `Ident`s?